### DISCUSSION TITLE
Old Hud/UI question

### DISCUSSION THREAD

**Discussion by CRRPPlasmex** [Courseplay/courseplay#7656] [Ideas]
Hey there Devs of Courseplay :) 

Ive enjoyed your mod now in LS19 for a decade and iam wondering if it would be possible to bring the old stlye back like it was in Ls19 since i personally find the new LS22 Version not quite fitting in terms of usage. In my opinion Courseplay was easier to use in LS19. I mean yeah its an early version but do you guys consider in bringing the old style back ? :) 

Greetings from Germany and nice job done with courseplay ;)

**Comment by Tensuko** [comment]
Also das alte HUD wird nicht wieder kommen, aber wir lesen die Wünsche dazu natürlich natürlich. Wir überlegen uns etwas.

**Reply by CRRPPlasmex** [reply]
wäre echt nice wenn ich sag mal zumindestens etwas in der art kommt da das rumgeklicke im menü mir persönlich etwas auf die Nerven geht :D 

allerdings muss ich sagen das ich den neuen CP noch nicht ganz gepeilt habe. 

aber wie gesagt wenn sowas in der art alter Menü/hud kommen würde, würde ich und bestimmt auch paar andere sich sehr freuen mal schauen was kommt :D
